pipeline-runner/R/gem2s-2-load_cellranger_files.R

```diff
@@ -0,0 +1,66 @@
+#' Read input folder of 10x data
+#'
+#' @inheritParams download_cellranger
+#' @param prev_out list with experiment configuration settings
+#'
+#' @return list with 'output' slot containing \itemize{
+#'   \item{"counts_list"}{named list of dgCMatrix per sample}
+#'   \item{"annot"}{data.frame from reading features.tsv.gz for each sample}
+#' }
+#' @export
+#'
+load_cellranger_files <- function(input, pipeline_config, prev_out) {
+  message("Loading cellranger output ...")
+
+  # destructure previous output
+  config <- prev_out$config
```

Comment:
what happens here if `config` is not in `prev_out`?

Reply:
`call_read10x` will fail

Reply:
so let's add a guard here to check if `config` is there before we try to access it, if it is not there, then raise an error ourselves.